src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceContacts.java

```diff
@@ -0,0 +1,183 @@
+    }
+
+    /**
+     * Method to get "edit" button.
+     * @return web element edit button.
+     */
+    public WebElement getEditButton() {
+        return this.webDriver.findElement(this.editContactButton);
+    }
+    /**
+     * Method to get "save" button.
+     * @return web element save button.
+     */
+    public WebElement getSaveContactButton() {
+        return this.webDriver.findElement(this.saveContactButton);
+    }
+    /**
+     * Method to know if an account is saved.
+     * @return the new contact name displayed in label.
+     */
+    public String newContactSavedName() {
+        return this.webDriver.findElement(this.newContactLabel).getText();
+    }
+    /**
+     * Method to know if an account was delete.
+     * @return the contact home page.
+     */
+    public String contactHomePage() {
+        wait = DriverManager.getInstance().getWaiter();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(this.homeContact));
+        return this.webDriver.findElement(this.homeContact).getText();
+    }
+
+    /**
+     * Method to get the last concat name link.
+     * @return the last contact.
+     */
+    public WebElement getContactNameLink() {
+        return this.webDriver.findElement(this.lastContactLink);
+    }
+    /**
+     * Method to get "delete" selected contact.
+     * @return delete button.
+     */
+    public WebElement getDeleteButton() {
+        return this.webDriver.findElement(this.deleteContactButton);
+    }
+
+    /**
+     * Method to edit contact fields.
+     * @param contactName the contact name.
+     */
+    public void editContact(final String contactName) {
+        setContactNameTextField(contactName);
+    }
+
+    /**
+     * Method to create a new contact.
+     * @param contactName the contact name.
+     * @param contactLastName the contact name.
+     */
+    public void fillInfoNewContact(final String contactName, final String contactLastName) {
+        setContactNameTextField(contactName);
+        setContactLastNameTextField(contactLastName);
+    }
+
+    /**
+     * Method to set contactName attribute.
+     * @param contactName string value.
+     */
+    public void setContactNameTextField(final String contactName) {
+        webDriver.findElement(this.contactNameTextField).sendKeys(contactName);
+    }
+
+    /**
+     * Method to set contact las Name attribute.
+     * @param contactLastName string value.
+     */
+    public void setContactLastNameTextField(final String contactLastName) {
+        webDriver.findElement(this.contactLastNameTextField).sendKeys(contactLastName);
+    }
+     /**
+     * Method to do click "delete" selected contact.
+     */
+    public void clickDeleteContact() {
+        getDeleteButton().click();
+    }
+    /**
+     * Method to do click accept button on the confirm alert before delete contact.
+     */
+    public void clickDeleteAlert() {
+        wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)